deployer/deployer.py

```diff
@@ -330,19 +330,19 @@ def start_deployment(self, data, version):
                     print("Autoscaling error")
         return success
 
-    def check_task_creation(self, id_task):
-        retries = 200
+    def wait_until_task_finish(self, id_task):
+        retries = 0
         wait_seconds = 10
         url = "http://{0}".format(self.asgard_base_url)
-        for r in range(retries):
+        status = ""
+        while retries < 200 and not (status == "completed" or status == "failed"):
```

Comment:
`retries` variable doesn't increase, and I also thing it `tries` or `attempts` would be better, or change to `retries_left` but initialized to 200 and decrement it.